__test__/pet.test.js

```diff
@@ -0,0 +1,156 @@
+const Pet = require("../src/pet");
+
+describe("constructor", () => {
+  test("returns an object", () => {
+    expect(new Pet("Fido")).toBeInstanceOf(Object);
+  });
+  test("sets the name property", () => {
```

Comment:
Its common convention to leave a clear line between each test to help with readability, should add one in here